persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -205,60 +182,20 @@ private void generateSchemaBalFile(Path persistPath, String packageName) throws
         }
     }
 
-    private void updateBallerinaToml(List<String> schemas) throws BalException {
+    private void updateBallerinaToml(List<String> schemas, String module, String datastore) throws BalException {
         try {
-            SyntaxTree syntaxTree = TomlSyntaxGenerator.updateBallerinaToml(
-                    Paths.get(this.sourcePath, BALLERINA_TOML), schemas);
-            writeOutputSyntaxTree(syntaxTree,
+            String syntaxTree = TomlSyntaxGenerator.updateBallerinaToml(
+                    Paths.get(this.sourcePath, BALLERINA_TOML), schemas, module, datastore);
+            writeOutputString(syntaxTree,
                     Paths.get(this.sourcePath, BALLERINA_TOML).toAbsolutePath().toString());
-            errStream.println("Updated Ballerina.toml with database configurations.");
-        } catch (Exception e) {
-            throw new BalException("could not update the Ballerina.toml with database configurations . " +
-                    e.getMessage());
-        }
-    }
-
-    private void createConfigTomlFile(List<String> schemas, String packageName) throws BalException {
-        try {
-            Path configPath = Paths.get(this.sourcePath, CONFIG_SCRIPT_FILE).toAbsolutePath();
-            SyntaxTree syntaxTree = TomlSyntaxGenerator.createConfigToml(schemas, packageName);
-            writeOutputSyntaxTree(syntaxTree, configPath.toString());
-            errStream.println("Created Config.toml file inside the Ballerina project.");
+            errStream.println("Updated Ballerina.toml with persist configurations : 'datastore = \"mysql\"', " +
+                    String.format("'module = %s'.", module));
         } catch (Exception e) {
-            throw new BalException("could not add Config.toml file inside the Ballerina project. " +
+            throw new BalException("could not update the Ballerina.toml with database configurations. " +
```

Comment:
```suggestion
            throw new BalException("could not update the Ballerina.toml with persist configurations. " +
```

Reply:
updated in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)